sale_force_invoiced/model/sale_order.py

```diff
@@ -16,7 +16,7 @@ class SaleOrder(models.Model):
                                          'pending to invoice.',
                                     readonly=True,
                                     states={'done': [('readonly', False)]},
-                                    default=False)
```

Comment:
Can you add `copy=False` 

Otherwise, you will never be able to invoice the copied sale order as it is already fully invoiced.